### ISSUE TITLE
evaluation categories in object detection

### ISSUE THREAD

**Issue by youthHan** [epic-kitchens/annotations#23]
Hi, thank you for the charming work of the dataset.
I have a question about the classes evaluated:
The class_map provided in your object detection pre-trained model has only 290 catogories. So I am confused about what collection of categories would be evaluated.

Ref:

Thank you for your interest in the dataset. Your model should detect all **clustered** noun classes which appear in the training set (of which there are 295). You can find the noun classes for each bounding box in the `noun_class` column of the `EPIC_train_object_labels.csv`. The class label and names of objects contained in each class can be found in `EPIC_noun_classes.csv`.

_Originally posted by @hazeld in https://github.com/epic-kitchens/annotations/issues/14#issuecomment-513697964_
